educe/rst_dt/corpus.py

```diff
@@ -36,9 +35,20 @@ class Reader(educe.corpus.Reader):
     def __init__(self, corpusdir):
         educe.corpus.Reader.__init__(self, corpusdir)
 
-    def files(self):
+    def files(self, exclude_file_docs=False):
+        """
+        Parameters
+        ----------
+        exclude_file_docs : boolean, optional (default=False)
+            If True, fileX documents are ignored. The figures reported by
+            (Li et al., 2014) on the RST-DT corpus indicate they exclude
+            fileN files, whereas Joty seems to include them.
+            fileN documents are more damaged than wsj_XX documents, e.g.
+            text mismatches with the corresponding document in the PTB.
+        """
```

Comment:
I tried to loosely follow the numpydoc convention used in numpy, scipy and the scikits:
https://github.com/numpy/numpy/blob/master/doc/HOWTO_DOCUMENT.rst.txt

I agree that the current design and API would have called for an implementation around is_interesting, but I ultimately chose a more direct and local (aka hardcoded) implementation.
`fileN` documents form a "stable" subset of the corpus, that has its own peculiarities and is excluded from some experiments, so direct hardcoding seemed to make sense to me.

Last, you are right in that the extractor should expose and delegate this parameter.
I had been preparing this PR for too long and just wanted to kick it out so that we can move on incrementally, afterwards.